langchain/document_transformers/__init__.py

```diff
@@ -0,0 +1,23 @@
+from langchain.document_transformers.embeddings_redundant_filter import (
+    EmbeddingsClusteringFilter,
+    EmbeddingsRedundantFilter,
+    _DocumentWithState,
+    _filter_similar_embeddings,
+    _get_embeddings_from_stateful_docs,
+    get_stateful_documents,
+)
+from langchain.document_transformers.text_extract import DoctranPropertyExtractor
+from langchain.document_transformers.text_qa import DoctranQATransformer
+from langchain.document_transformers.text_translate import DoctranTextTranslator
+
+__all__ = [
+    "DoctranQATransformer",
+    "DoctranTextTranslator",
+    "DoctranPropertyExtractor",
+    "EmbeddingsClusteringFilter",
+    "EmbeddingsRedundantFilter",
+    "_DocumentWithState",
```

Comment:
no need to export private stuff

Reply:
ya, legacy from the initial commit. pulling them out now.

Reply:
I did so because previously I was running into failures because there are tests that import these internal methods and classes, e.g [test_embeddings_filter.py](https://github.com/hwchase17/langchain/blob/master/tests/integration_tests/retrievers/document_compressors/test_embeddings_filter.py)

But not sure if there's another way to run the tests that makes these methods available. Should I go ahead and remove them?

Reply:
ya, eh may need them:
```
tests/unit_tests/test_document_transformers.py:2: error: Module "langchain.document_transformers" has no attribute "_filter_similar_embeddings"  [attr-defined]
langchain/retrievers/document_compressors/embeddings_filter.py:8: error: Module "langchain.document_transformers" has no attribute "_get_embeddings_from_stateful_docs"  [attr-defined]
langchain/retrievers/document_compressors/embeddings_filter.py:8: error: Module "langchain.document_transformers" has no attribute "get_stateful_documents"  [attr-defined]
```

Reply:
we should just update those important statements to import from the specific files instead of from document_transformers init

Reply:
ya ill clean this all up.